iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/VariantManagerImplTest.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Comment:
Fix copyright information

Reply:
I juste renamed the class, it is not fair to change the copyright.